tesla_http_proxy/rootfs/app/run.sh

```diff
@@ -89,10 +89,24 @@ done
 
 # verify public key is accessible with valid TLS cert
 bashio::log.info "Testing public key..."
-if ! curl -sfD - "https://$DOMAIN/.well-known/appspecific/com.tesla.3p.public-key.pem"; then
-  bashio::log.fatal "Fix public key before proceeding."
-  exit 1
-fi
+set +e
```

Comment:
I don't like changing `set` midway through a script, it makes debugging more confusing.  Instead, can you add `|| true` to the end of curl?

Reply:
I actually thought about this after the last commit. Will change it.